django/cantusdb_project/main_app/permissions.py

```diff
@@ -167,3 +168,15 @@ def user_can_manage_source_editors(user: User) -> bool:
         or user.is_staff
         or user.groups.filter(name="project manager").exists()
     )
+
+
+def project_manager_check(user: User) -> bool:
```

Comment:
Great. Can we just rename it similarly to other functions in this file? Something like `user_is_project_manager`.

Reply:
Yep...and I'll fix the failing test.